src/App.tsx

```diff
@@ -1,17 +1,100 @@
-import React from 'react';
-import './App.css';
+import { FC, useState } from 'react';
+import './App.scss';
 
+import todos from './api/todos';
 import users from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos: Todo[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
+
+const App: FC = () => {
+  const [inputedTitle, setInputedTitle] = useState('');
+  const [choosedUser, setCoosedUser] = useState('');
```

Comment:
use more generic names
```suggestion
  const [title, seTitle] = useState('');
  const [userId, setUserId] = useState('');
```